src/tests/uat/vulnerability_assessment_uat.py

```diff
@@ -233,17 +258,28 @@ def main():
     print(25 * ' ', 'Vulnerability Level')
     print(SYMBOLS * DELIMITER)
 
-    api_results = get_vulnerability()
-    vulnerability = Vulnerability(cb, CVE_ID)
-    assert api_results == vulnerability._info, 'Test Failed Expected: {} Actual: {}'.\
-        format(api_results, vulnerability._info)
+    CVE_ID = 'CVE-2008-5915'
+
+    api_results = get_vulnerability(cve_id=CVE_ID)
+    vulnerability = None
+    try:
+        vulnerability = Vulnerability(cb, CVE_ID)
+    except MoreThanOneResultError as ex:
+        vulnerability = ex.results
+
+    if type(vulnerability) == list:
+        sdk_results = [vuln._info for vuln in vulnerability]
+    else:
+        sdk_results = vulnerability._info
+
+    assert api_results == sdk_results, 'Test Failed Expected: {} Actual: {}'.format(api_results, sdk_results)
     print('Get vulnerability details for a specific CVE ID.....................OK')
 
-    api_results = get_affected_devices(data={'os_product_id': '313_0'})['results']
-    query = vulnerability.affected_assets('313_0')
+    api_results = get_affected_devices(cve_id=CVE_ID, data={'os_product_id': vulnerability.os_product_id})['results']
+    query = vulnerability.get_affected_assets()
```

Comment:
When I ran the tests vulnerability is a list and as a result vulnerability.os_product_id and vulnerability.get_affected_assets() is raising an error AttributeError: 'list' object has no attribute 'xxx'

Reply:
I forgot that case in the follow up test. I've fixed it now